### PR TITLE
Doc: Updated broken relrefs to "Value mappings in topic "State timeline".

### LINKED ISSUE #36177: [Docs] feedback: /panels/visualizations/state-timeline.md
[The state timeline paragraphs ](https://grafana.com/docs/grafana/latest/panels/visualizations/state-timeline/#value-mappings) references a [Value Mapping](https://grafana.com/docs/grafana/latest/panels/visualizations/state-timeline/%20%7b%7b%20refref%20%22../value-mappings.md%22%7d%7d%20) link that returns a 404 when I try to follow it. I have found an article on [Value Mapping](https://grafana.com/docs/grafana/latest/panels/value-mappings/) within the `Panels` section that I believe is where it should land.

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
Fixes https://github.com/grafana/grafana/issues/36177.

### CODE REVIEW

File: docs/sources/panels/visualizations/state-timeline.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Controls the opacity of state regions.
 
 ## Value mappings
 
-To assign colors to boolean or string values, use the [Value mappings](< {{ refref "../value-mappings.md"}} >).
+To assign colors to boolean or string values, use [Value mappings]({{< relref "./value-mappings.md" >}}).
```
Comment (oddlittlebird) [inline docs/sources/panels/visualizations/state-timeline.md:44]: Wait, what? This change shouldn't work. Value mappings is up a level from the visualizations files, so it should require two dots, not one.

Comment (achatterjee-grafana) [reply docs/sources/panels/visualizations/state-timeline.md:44]: The older relref [Value mappings](< {{ refref "../value-mappings.md"}} >) did not work. Any suggestion?

**Comment by achatterjee-grafana** [comment]
Oh wait - I see what I did. Ugh! Will ask for re-review when fixed.

File: docs/sources/panels/visualizations/state-timeline.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/panels/visualizations/state-timeline.md]: I see it working in my local build. Maybe an issue with the quotes ...

**Review by oddlittlebird — APPROVED** [review]

**Pull request merged**